admin/app/view_models/workarea/admin/pricing_sku_view_model.rb

```diff
@@ -16,6 +16,10 @@ def product
       def inventory
         @inventory ||= Inventory::Sku.where(id: model.id).first
       end
+
+      def on_sale?
```

Comment:
This will display "On Sale: Yes" for SKUs in the index where any prices are marked as "On Sale".